src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,120 @@
+package duke.command;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents a command for adding tasks to the task lists. It comprises of 3 types
+ * of task to be created, namely todo, deadline and event.
+ */
+public class AddCommand extends Command {
+    private int taskType;
+    private String taskDetails;
+
+    /**
+     * Constructor to create an instance of AddCommand.
+     *
+     * @param taskType Integer value representing the type of task, 0-TODO Task,
+     *                 1-DEADLINE Task, 2-EVENT Task
+     * @param taskDetails String representation of the task description
+     * @return instance of AddCommand
+     */
+    public AddCommand(int taskType, String taskDetails) {
+        this.taskType = taskType;
+        this.taskDetails = taskDetails;
+    }
+
+    /**
+     * Create the corresponding task based on the taskType in AddCommand and adds task
+     * to the TaskList. The task list is then stored on a file under data/tasks.txt
+     *
+     * @param tasks a list to add and keep track of the tasks
```

Comment:
Should be capital letter after parameter, aka @param tasks List to add and keep track of the tasks